plutus/budget_manager/project_budget.py

```diff
@@ -0,0 +1,67 @@
+from constants import (
```

Comment:
`from constats import *` should do the same since you want all these constants anyway.